src/components/gamespage/GamesPageProgressSection/index.jsx

```diff
@@ -1,16 +1,33 @@
 import React from "react";
 import "./index.css";
 import { useTranslation } from "react-i18next";
+import i18next from "i18next";
 import { Progress, Row, Col } from "antd";
 import { QuestionOutlined } from "@ant-design/icons";
 
+const GRID_SPAN_OBJ = {
+  currLevelSpan: 7,
```

Comment:
you dont need to add Span to the end of everything if the constant name already has it. we know these are spans

but you can just call this GRID_SPANS

and write a comment above it.

Reply:
PTAL